config/manager/kustomization.yaml

```diff
@@ -16,4 +16,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: hashicorp/vault-secrets-operator
-  newTag: 0.1.0-beta
```

Comment:
This change keeps popping up in other PRs. I think it is a by product of running kustomize.

Reply:
This one is so annoying.. idk what to do with it, nearly every commit is getting this when we're working on CRDs at this point.